packages/calypso-e2e/src/data-helper.ts

```diff
@@ -93,7 +93,7 @@ export function toTitleCase( words: string[] | string ): string {
 	}
 
 	const result = words.map( function ( word ) {
-		return word.charAt( 0 ).toUpperCase() + word.slice( 1 ).toLowerCase();
+		return word.charAt( 0 ).toUpperCase() + word.slice( 1 );
```

Comment:
No longer lowercases non-head characters.

As a result, with the input `SEO (Manage)`, this function:

`Seo (manage)` --> `SEO (Manage)`